### PR TITLE
Improve CIFAR10 example

### PR DESCRIPTION
**Pull request by ghost**
- Extending the Jupyter Notebook to also reflect the distributed training and inference examples
- Correcting the description of attributes in experiment configurations
- Improved clarity of naming of `distributed_*` experiment configuration files

### CODE REVIEW

File: computer_vision/cifar10_pytorch/distributed.yaml [hunk 1]
```diff
@@ -1,19 +0,0 @@
-name: cifar10_pytorch_distributed
```
Comment (ioga) [inline computer_vision/cifar10_pytorch/distributed.yaml:1]: why did you delete these `distributed*` yamls?

Comment (ghost) [reply computer_vision/cifar10_pytorch/distributed.yaml:1]: Ah sorry, intended to rename but didn't commit the last change. Updated now. Thanks for the catch

Comment (ioga) [reply computer_vision/cifar10_pytorch/distributed.yaml:1]: I see. I'd suggest keeping `distributed.yaml` as is and not renaming it. almost all other examples use this name, because we assume that training is the default mode. it's really abnormal to have a `distributed_somethingelse.yaml`, like this example does.

Comment (ghost) [reply computer_vision/cifar10_pytorch/distributed.yaml:1]: Sure, that makes sense. I've undone the renaming of that file/experiment.

**Review by ioga — APPROVED** [review]
lgtm
